pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = [
   { name = "Mike Oakley", email = "mike-oakley@users.noreply.github.com" },
 ]
 readme = "README.md"
-requires-python = ">=3.8"
+requires-python = ">=3.9"
```

Comment:
Pydantic v2.12 seems allergic to Python < 3.9.

Reply:
Python 3.8 support has been dropped in [v2.11.0](https://github.com/pydantic/pydantic/releases/tag/v2.11.0) with https://github.com/pydantic/pydantic/pull/11258